src/System.Private.ServiceModel/tests/Common/Infrastructure/src/ConditionalWcfTest.cs

```diff
@@ -153,10 +153,8 @@ public static bool SPN_Available()
         // Returns 'true' if the server is configured to accept client certificates.
         public static bool Server_Accepts_Certificates()
         {
-            // Temporarily use the simple heuristic that if we are running the services locally, it does.
-            // Refactor this after integration to address https://github.com/dotnet/wcf/issues/1024 
-            return GetConditionValue(nameof(Server_Accepts_Certificates),
-                                     Server_Is_LocalHost);
+            //Both IIS hosted and self hosted servers accept server certificate now
+            return true;
```

Comment:
Agree we should remove it as it's no longer needed. I will do it separately after this PR gets in. This PR itself is a non trivial efforts to test. Thus I prefer do the clean up later on.